test/index.html

```diff
@@ -1,97 +0,0 @@
-<!DOCTYPE HTML>
```

Comment:
This file had each test individually listed. Is there a good way to pull in all the test files without manually listing them? I guess we could always browserify them into one file, but we probably don't want to add that to the main watch task.